tests/sles4sap/microsoft_sdaf/sdaf_cleanup.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+# Summary: Triggers cleanup of the workload zone and SUT using SDAF automation.
+# It also removes all SDAF test related files from deployer VM.
+# Post run hooks are generally disabled during normal module run so the infrastructure persists between test modules.
+# Cleanup is triggered only with B<SDAF_DO_CLEANUP> set to true, which is done by scheduling this module at the end of test flow.
+
+use parent 'sles4sap::microsoft_sdaf_basetest';
+use strict;
+use testapi;
+use warnings;
+use sles4sap::sdaf_library;
+
+sub test_flags {
+    return {fatal => 1};
+}
+
+sub run {
+    serial_console_diag_banner('end: sdaf_cleanup.pm');
+    # Setting up cleanup flag will trigger post run cleanup
+    set_var('SDAF_DO_CLEANUP', 'YES! Please, do it.');
```

Comment:
is there a place it is ...?
```
set_var('SDAF_DO_CLEANUP', undef);
```

Reply:
There's a `set_var` here and an `unless` in the hook. Sound like sort of "double negation".

Is it possible to do the same just by calling 
```
# Cleanup SDAF files form Deployer VM
    connect_target_to_serial();
    cleanup_sdaf_files();
    disconnect_target_from_serial();
```
here and leave the hook mostly empty?

Reply:
Hmm I wanted to keep all the cleanup routines in the same file so one does not have to search in multiple places... but I'll try it according to you suggestion and see if it fits with upcoming test modules.